modules/logging/cores.go

```diff
@@ -0,0 +1,35 @@
+package logging
+
+import (
+	"github.com/caddyserver/caddy/v2"
+	"github.com/caddyserver/caddy/v2/caddyconfig/caddyfile"
+	"go.uber.org/zap/zapcore"
+)
+
+func init() {
+	caddy.RegisterModule(MockCore{})
+}
+
+// MockCore for testing
```

Comment:
These godoc comments end up in https://caddyserver.com/docs/json/ so giving a clearer warning to not use it is a good idea I think.

```suggestion
// MockCore is a no-op module, purely for testing
// during development. Should not be used in apps.
```